### PR TITLE
First version of CPV detector

### PR DESCRIPTION
**Pull request by peressounko**
Frirst version of CPV with hit creation is implemented

### CODE REVIEW

**Comment by peressounko** [comment]
Looks like default clang-format file does not correspond to what is expected by Travis. Which file should be used? Is it possible to update in in O2 distribution?
     Thanks, Dmitri

**Comment by sawenzel** [comment]
The file used is the one from the repository. What is your clang-format version?

**Comment by peressounko** [comment]
clang-format version 7.0.0 (tags/RELEASE_700/final), command: find . -iname "*.h" -o -iname "*.cxx" | xargs clang-format -style=file -i

**Comment by shahor02** [comment]
I had similar problem, it appeared that different system-provided version was used. I had to uninstall it and use the one from the aliBuild. Could you do `which clang-format`?

**Comment by peressounko** [comment]
[Clang/latest-o2] ~/alice/O2 $> which clang-format
/home/prsnko/alice/sw/ubuntu1804_x86-64/Clang/7.0.0-1/bin/clang-format

**Comment by peressounko** [comment]
Found reason for clang fail: I did not apply formatting to *.C files

**Comment by shahor02** [comment]
You can do simply
git clang-format HEAD^n
with n being the number of commits to cover.

**Comment by peressounko** [comment]
Dear Ruben, Sandro, what is tested in 3 failed tests? I do not see anything related to CPV... Thanks, Dmitri

**Comment by shahor02** [comment]
Hi @peressounko ,

What is test is just macro loading w/o execution, e.g. `root  -n -b -l -q -e '.L drawCPVgeometry.C++'
It looks like you are missing some headers.

**Comment by peressounko** [comment]
Thank you, I fixed macros, but still do not see how the reason of failure of build/O2/o2-dev-fairroot is related to CPV

**Comment by shahor02** [comment]
o2-dev-fairroot always fails and is not required for the merging.

File: Detectors/CPV/base/include/CPVBase/Digit.h [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  int getAbsId() const { return mAbsId; }
+  void setAbsId(int cellId) { mAbsId = cellId; }
+
+  /// \brief Energy deposited in a cell
+  double getAmplitude() const { return mAmplitude; }
+  void setAmplitude(double amplitude) { mAmplitude = amplitude; }
+
+  Label getLabel(int idx) const
+  {
+    if (idx < kMaxLabels)
```
Comment (sawenzel) [inline Detectors/CPV/base/include/CPVBase/Digit.h]: This is a nice PR and we can merge it very soon. Just one little detail: Could you please **systematically** add `{}` for `if-else` statements (according to coding rules). This concerns a couple of places.

**Pull request merged**